src/core/server/graph/resolvers/User.ts

```diff
@@ -26,15 +24,15 @@ const maybeLoadOnlyIgnoredUserID = (
     return [];
   }
 
-  // Get the field names of the fields being requested, if it's only the ID,
-  // we have that, so no need to make a database request.
-  const fields = getRequestedFields<GQLUser>(info);
-  if (fields.length === 1 && fields[0] === "id") {
-    return users.map(({ id }) => ({ id }));
-  }
```

Comment:
This is an important optimization that should not be removed!

Reply:
@wyattjoh I realized that we actually don't have to remove this optimization (the error doesn't arise from returning id's of nonexistent users, just from literally returning `null`) and have re-added it